staging/src/k8s.io/apiserver/pkg/server/filters/priority-and-fairness.go

```diff
@@ -78,6 +76,9 @@ func WithPriorityAndFairness(
 		klog.Warningf("priority and fairness support not found, skipping")
 		return handler
 	}
+	waitingMark.readOnlyObserver = fcmetrics.GetReadOnlyWaitingConcurrency()
```

Comment:
This seems to be causing a race in our tests:
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/110164/pull-kubernetes-unit/1550364149416988672

Reply:
Oh, I did not think that writes of the same value would be a problem.  Easily fixed...